frontend/packages/client/src/data/contractsAndPaths.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "name": "$FLOW",
```

Comment:
why the "$FLOW", "$BLT" instead of just FLOW, BLT etc?

Reply:
I actually requested that at https://github.com/onflow/CAST/pull/10#pullrequestreview-1593587199 just because usually crypto users identify fungible tokens better like that, however we can just leave it as FLOW, BLT etc. for consistency. What do you think @Parikshit85 @vishalchangrani ?

Reply:
We've added these considering the UX as tokens are usually identifiable via the '$' sign. @vishalchangrani .
However we can remove them for keeping the standard view.

Reply:
@Parikshit85 lets proceed by omitting the **$** in order to be consistent 🙏